express-app/src/config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+    secretKey: process.env.SECRET_KEY || 'defaultSecretKey',
```

Comment:
Should this be BIG_SECRET to align with Docker compose name?

Reply:
Umm not sure, will have to check.